src/Components/DnDLayout/GridLayout.tsx

```diff
@@ -42,6 +53,26 @@ const getResizeHandle = (resizeHandleAxis: string, ref: React.Ref<HTMLDivElement
   );
 };
 
+const LayoutEmptyState = () => {
+  return (
+    <PageSection className="empty-layout pf-v5-u-p-0">
+      <EmptyState variant={EmptyStateVariant.lg} className="pf-v5-u-p-sm">
+        <EmptyStateHeader titleText="No dashboard content" headingLevel="h2" icon={<EmptyStateIcon icon={PlusCircleIcon} />} />
+        <EmptyStateBody>
+          You don’t have any widgets on your dashboard. To populate <br /> your dashboard, drag <GripVerticalIcon /> items from the blue widget bank
```

Comment:
@CodyWMitchell  I wouldn't use breaks here. It won't work at small resolutions 
![Screenshot 2024-04-02 at 10 49 20](https://github.com/RedHatInsights/widget-layout/assets/1287144/9c2b66b4-634d-4e85-bc47-8cab65ecb194)